src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerRemoveImageFunctionalTest.groovy

```diff
@@ -74,13 +81,20 @@ class DockerRemoveImageFunctionalTest extends AbstractFunctionalTest {
                 force = true
                 targetImageId { buildImage.getImageId() }
             }
+
+			task removeImageAndCheckRemoval(type: DockerListImages) {
+				dependsOn removeImage
+    			showAll = true
+    			filters = '{"dangling":["true"]}'
+			}
+            
         """
 
         when:
-        BuildResult result = build('removeImage')
+        BuildResult result = build('removeImageAndCheckRemoval')
 
         then:
-        result.output.contains("Removing image with ID")
+        !result.output.contains("repository")
```

Comment:
It would be better not to check console log but use [custom implementation of ResponseHandler](https://github.com/bmuschko/gradle-docker-plugin/blob/master/src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerListImages.groovy#L51) which will do the check.